scripts/Calibration/Examples/TubeCalibDemoMaps_All.py

```diff
@@ -500,8 +502,8 @@ def completeCalibration(filename):
     mantid.ApplyCalibration(Workspace=CalibInstWS, PositionTable=calibrationTable)
 
     # == Save workspace ==
-    # SaveNexusProcessed( CalibInstWS, path+'TubeCalibDemoMapsResult.nxs',"Result of Running TCDemoMaps.py")
-    # print "saved calibrated workspace (CalibInstWS) into Nexus file TubeCalibDemoMapsResult.nxs"
+    # mantid.SaveNexusProcessed(CalibInstWS, path+'TubeCalibDemoMapsResult.nxs', "Result of Running TCDemoMaps.py")
```

Comment:
Usually I would remove these leftovers but this is some example/tutorial and this is quick-and-dirty way to switch the file I/O on and off. I wasn't sure if this would be needed for pedagogical purposes so I chose to keep the lines.

The commented lines can be removed if you so wish, but what to do with the same but uncommented code in some other files in `scirpts/Calibration/Examples/`?